ui/app/templates/vault/cluster/access/oidc.hbs

```diff
@@ -35,7 +33,7 @@
   {{#unless this.isCta}}
     {{! show tab links in list routes }}
     <div class="tabs-container box is-sideless is-fullwidth is-paddingless is-marginless" data-test-oidc-tabs>
-      <nav class="tabs">
+      <nav class="tabs" aria-label="navigation for managing OIDC">
```

Comment:
I suspect, and from what I can read online that screen readers cannot read acronyms. Meaning OIDC will be read as a word. Do you think in these cases its worth writing Open ID Connect instead?

Reply:
Great point, and you are correct! Thanks, I'll fix that -- let me know if you catch any others